### PR TITLE
colordiff: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/colordiff.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# colordiff
+
+> A tool to colorize diff diff output.
+> The Perl script colordiff is a wrapper for `diff` and produces the same output but with pretty 'syntax' highlighting. Colour schemes can be customized..
```
Comment (agnivade) [inline pages/common/colordiff.md:4]: Any reason syntax is in quotes? Also, let's remove "pretty" to remain objective. And the last sentence has 2 periods.

File: pages/common/colordiff.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> More information: <https://github.com/kimmel/colordiff>.
+
+- Compare files:
+
+`colordiff {{file-1}} {{file-2}}`
```
Comment (agnivade) [inline pages/common/colordiff.md:9]: We usually use snake_case. Or just file1, file2

File: pages/common/colordiff.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Compare files, output in two columns:
+
+`colordiff -y {{file-1}} {{file-2}}`
+
+- Compare files, Ignore case differences in file contents:
+
+`colordiff -i {{file-1}} {{file-2}}`
+
+- Compare files,  Report when two files are the same:
```
Comment (agnivade) [inline pages/common/colordiff.md:19]: We can just remove "Compare files," from this example onwards. After 3 times, it becomes implicit.

File: pages/common/colordiff.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Compare files:
+
+`colordiff {{file-1}} {{file-2}}`
+
+- Compare files, output in two columns:
+
+`colordiff -y {{file-1}} {{file-2}}`
+
+- Compare files, Ignore case differences in file contents:
```
Comment (agnivade) [inline pages/common/colordiff.md:15]: How about- "Compare files ignoring case differences in file contents"

**Review by agnivade — CHANGES_REQUESTED** [review]
Thanks for the PR. Please take a look at the comments.

File: pages/common/colordiff.md (hunk 1 above)
Comment (thamaraiselvam) [reply pages/common/colordiff.md]: No reason, I will remove it

File: pages/common/colordiff.md (hunk 4 above)
Comment (thamaraiselvam) [reply pages/common/colordiff.md]: it make sense and I have removed "compare files" content for all examples except first.

**Review by agnivade — APPROVED** [review]
Thanks!

File: pages/common/colordiff.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# colordiff
+
+> A tool to colorize diff diff output.
```
Comment (navarroaxel) [inline pages/common/colordiff.md:3]:
```suggestion
> A tool to colorize diff output.
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Pull request merged**